### PR TITLE
Dump labels and annotations for function via getmeta cli

### PR DESCRIPTION
**Pull request by nmiletic**
## Description
`fission function getmeta` was only showing name and environment of the function. This change adds Annotations and Labels.

## Which issue(s) this PR fixes:

Fixes #

## Checklist:

- [ ] I ran tests as well as code linting locally to verify my changes. 
- [X ] I have done manual verification of my changes, changes working as expected.
- [ ] I have added new tests to cover my changes.
- [ ] My changes follow contributing guidelines of Fission.
- [ ] I have signed all of my commits.

### CODE REVIEW

**Comment by sanketsudake** [comment]
@nmiletic could you please share sample output before and after your change ?

**Comment by nmiletic** [comment]
> @nmiletic could you please share sample output before and after your change ?

Bofore:

**Review by shubham-bansal96 — APPROVED** [review]

File: pkg/fission-cli/cmd/function/getmeta.go [hunk 1]
```diff
@@ -46,10 +44,16 @@ func (opts *GetMetaSubCommand) do(input cli.Input) error {
 		return errors.Wrap(err, "error getting function")
 	}
 
-	w := tabwriter.NewWriter(os.Stdout, 0, 0, 1, ' ', 0)
-	fmt.Fprintf(w, "%v\t%v\n", "NAME", "ENV")
-	fmt.Fprintf(w, "%v\t%v\n", fn.ObjectMeta.Name, fn.Spec.Environment.Name)
-	w.Flush()
+	fmt.Printf("Name: %v\n", fn.ObjectMeta.Name)
+	fmt.Printf("Environment: %v\n", fn.Spec.Environment.Name)
+	fmt.Println("Labels:")
```
Comment (sanketsudake) [inline pkg/fission-cli/cmd/function/getmeta.go:49]: Should we avoid printing this string if labels is empty ?

File: pkg/fission-cli/cmd/function/getmeta.go [hunk 2]
```diff
@@ -46,10 +44,16 @@ func (opts *GetMetaSubCommand) do(input cli.Input) error {
...
+	for k, v := range fn.ObjectMeta.Labels {
+		fmt.Printf("  %s=%s\n", k, v)
+	}
+	fmt.Println("Annotations:")
```
Comment (sanketsudake) [inline pkg/fission-cli/cmd/function/getmeta.go:53]: Should we avoid printing this string if annotations is empty ?

**Review by sanketsudake — CHANGES_REQUESTED** [review]

File: pkg/fission-cli/cmd/function/getmeta.go (hunk 2 above)
Comment (nmiletic) [reply pkg/fission-cli/cmd/function/getmeta.go]: Is it ok list this? I am new to both go and PR process so excuse me if any mistakes.

**Comment by sanketsudake** [comment]
@mergifyio rebase

**Review by sanketsudake — APPROVED** [review]

**Comment by sanketsudake** [comment]
Thanks @nmiletic for you contribution 🎉

**Pull request merged**